apps/server/src/modules/board/gateway/dto/move-card-to-board.message.param.ts

```diff
@@ -0,0 +1,16 @@
+import { IsBoolean, IsMongoId, IsOptional } from 'class-validator';
+
+export class MoveCardToBoardMessageParams {
+	@IsMongoId()
+	cardId!: string;
+
+	@IsMongoId()
+	fromColumnId!: string;
+
+	@IsMongoId()
+	toColumnId!: string;
+
+	@IsOptional()
+	@IsBoolean()
+	forceNextTick?: boolean;
```

Comment:
looks like FE logic, having a hard time understanding why this belongs in BE

Reply:
The `forceNextTick` is initiated in the FE, the passed to the server and back to all clients of the respective channels. So it can be seen as actually belonging to the FE.